<!DOCTYPE html>
<html>

  <head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <title>Ribo centre - Application</title>
  <meta name="description" content="Ribo centre -- Application">
  <link rel="stylesheet" href="http://localhost:4000/css/main.css">
  <link rel="canonical" href="http://localhost:4000/application/">
<link rel="shortcut icon" type ="image/x-icon" href="http://localhost:4000/images/favicon.ico">



</head>


  <body>

    <div class="navbar-expand-lg navbar-dark" style='background-color: #333333; border-radius: 0; margin-bottom: 0; border: 0; height: 30px;' role="navigation">
  <div class="container-fluid" style='height: 30px; max-width: 1000px;'>
	  <ul class="nav navbar-nav navbar-right" style='height: 30px;'>
		<li><a style='display: inline; padding: 15px 15px 5px 5px; line-height: 30px;' class="nav-link-0" href="http://localhost:4000/">
			<img src="http://localhost:4000/images/header/home.svg" style='height: 20px; width: auto; margin: 0;'>
			Home</a></li>
		<li><a style='display: inline; padding: 15px 15px 5px 5px; line-height: 30px;' class="nav-link-0" href="http://localhost:4000/database">
			<img src="http://localhost:4000/images/header/database.svg" style='height: 20px; width: auto; margin: 0;'>
			Database</a></li>
		<li><a style='display: inline; padding: 15px 15px 5px 5px; line-height: 30px;' class="nav-link-0" href="http://localhost:4000/research">
			<img src="http://localhost:4000/images/header/research.svg" style='height: 20px; width: auto; margin: 0;'>
			Research</a></li>
		<li><a style='display: inline; padding: 15px 15px 5px 5px; line-height: 30px;' class="nav-link-0" href="http://localhost:4000/aboutus">
			<img src="http://localhost:4000/images/header/aboutus.svg" style='height: 20px; width: auto; margin: 0;'>
			About us</a></li>
		<li><a style='display: inline; padding: 15px 15px 5px 5px; line-height: 30px;' class="nav-link-0" href="https://gzlab.ac.cn/">
			GZNL-RDC
			<img src="http://localhost:4000/images/header/gznl2.svg" style='height: 20px; width: auto; margin: 0;'>
			</a></li>
	  </ul>
  </div>
</div>

<!--
  <div class="search-container">
    <p class="search-text">Search for something:</p>
    <div class="search-box">
      <input type="text" placeholder="Search...">
    </div>
    <p class="additional-text">Additional text here...</p>
  </div>

  <div class="background-image">
    <img src="http://localhost:4000/images/Picture-1.jpg" style='width: 100%; border-radius: 0; margin-top: 0; margin-bottom: 0;'>
  </div>
-->

<div class="full-width-image">
  <img src="http://localhost:4000/images/Picture-1.jpg" class="img-fluid" alt="Responsive image" style='width: 100%; border-radius: 0; margin-top: 0; margin-bottom: 0;'>
</div>



<div class="navbar navbar-expand-lg navbar-dark bg-primary" style='border-radius: 0; border: 0;' role="navigation">

  <div class="container-fluid" style='max-width: 1200px;'>
	<div class="navbar-header">
	  <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar-collapse-1" aria-expanded="false">
		<span class="sr-only">Toggle navigation</span>
		<span class="icon-bar"></span>
		<span class="icon-bar"></span>
		<span class="icon-bar"></span>
	  </button>
	
    <a class="navbar-brand" href="http://localhost:4000/">
		
		<img class="navbar-brand" src="http://localhost:4000/images/header/logo.svg" style='height:50px;margin-top:-14px;margin-bottom:0;padding:2px 2px;border-radius: 0;' alt="logo">
		<b style='font-size: 32px; color: #eeeeee;'>Ribocentre</b>
	</a>
	</div>
	<div class="collapse navbar-collapse" id="navbar-collapse-1">
	  <ul class="nav navbar-nav navbar-right">
		<li><a class="nav-link-1" href="http://localhost:4000/">Home</a></li>
		<li><a class="nav-link-1" href="http://localhost:4000/ribozyme">Ribozyme</a></li>
		<li><a class="nav-link-1" href="http://localhost:4000/structure">Structure</a></li>
		<li><a class="nav-link-1" href="http://localhost:4000/publications">Publications</a></li>
		<li><a class="nav-link-1" href="http://localhost:4000/team">Team</a></li>
		<li><a class="nav-link-1" href="http://localhost:4000/resources">Resources</a></li>
	  </ul>
	</div>
  </div>
</div>


    <div class="container-fluid">
      <div class="row">
        <div id="picid" class="col-sm-12">
  <h1 id="application">Application</h1>

<p><br /></p>

<blockquote>
  <p>Twister</p>
</blockquote>

<p>To establish RNA aptamers with high stability and expression levels, Jaffrey’s group designed the Tornado (twister-optimized RNA for durable overexpression) expression system for rapid RNA circularization. They use a P3 twister U2A ribozyme and a P1 twister ribozyme to generate RNAs of interest containing a 5’ hydroxyl and 2’,3’-cyclic phosphate. Then the 5’ and 3’ ends would become substrates for RtcB-mediated ligation.Finally, the RNAs of interest would be circularized.</p>

<p><img src="https://www.ribocentre.org/images/ApplyPic/twister.png" alt="drawing" style="weight:450px;height:600px;display:block;margin:0 auto;" /><br />
<a href="https://www.ribocentre.org/downloads/Twister.pdf" target="_blank" download="Appli-Twister.pdf"><button class="btn btn-secondary"><span class="glyphicon glyphicon-download-alt"></span>  Download Poster</button></a><br /><br />
<img src="https://www.ribocentre.org/images/ApplyPic/twister-ribozyme.png" alt="drawing" style="weight:450px;height:460px;display:block;margin:0 auto;" /><br />
<a href="https://www.ribocentre.org/downloads/twister-ribozyme-application.pdf" target="_blank" download="Appli-Ribozyme.pdf"><button class="btn btn-secondary"><span class="glyphicon glyphicon-download-alt"></span>  Download Poster</button></a><br /><br /></p>

<blockquote>
  <p>HDV</p>
</blockquote>

<p>Schematic diagram for simple and effective generating in vitro transcripts with homogenous 5’ ends and 3’ ends. As show in this diagram, we use hammerhead and HDV ribozymes to produce homogenous RNA with any 5’- and 3’- terminal sequence.</p>

<p><img src="https://www.ribocentre.org/images/ApplyPic/HDV.png" alt="drawing" style="weight:450px;height:600px;display:block;margin:0 auto;" /><br />
<a href="https://www.ribocentre.org/downloads/HDV.pdf" target="_blank" download="Appli-HDV.pdf"><button class="btn btn-secondary"><span class="glyphicon glyphicon-download-alt"></span>  Download Poster</button></a><br /><br /></p>

<p><img src="https://www.ribocentre.org/images/ApplyPic/hammerhead.png" alt="drawing" style="weight:350px;height:300px;display:block;margin:0 auto;" /><br />
<a href="https://www.ribocentre.org/downloads/Hammerhead-application.pdf" target="_blank" download="Appli-Hammerhead.pdf"><button class="btn btn-secondary"><span class="glyphicon glyphicon-download-alt"></span>  Download Poster</button></a><br /><br /></p>

<html>
<head>
	<meta http-equiv="Content-type" content="text/html; charset=utf-8" />
	<meta name="viewport" content="width=device-width,initial-scale=1,user-scalable=no" />
	<title>Ribozyme publication - export</title>
	<link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/1.12.1/css/jquery.dataTables.min.css" />
	<link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/buttons/2.2.3/css/buttons.dataTables.min.css" />
</head>
    <style>
		th {
        background-color: #0874c4;
        background-color: #0874c4;
        background-color: #0874c4;
        color: rgba(255,255,255,0.9);
		    cursor: pointer;
        }
	</style>
    <script type="text/javascript" src="https://code.jquery.com/jquery-3.5.1.js"></script>
	<script type="text/javascript" src="https://cdn.datatables.net/1.12.1/js/jquery.dataTables.min.js"></script>
	<script type="text/javascript" src="https://cdn.datatables.net/buttons/2.2.3/js/dataTables.buttons.min.js"></script>
	<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.1.3/jszip.min.js"></script>
	<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.1.53/pdfmake.min.js"></script>
	<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.1.53/vfs_fonts.js"></script>
	<script type="text/javascript" src="https://cdn.datatables.net/buttons/2.2.3/js/buttons.html5.min.js"></script>
	<script type="text/javascript" src="https://cdn.datatables.net/buttons/2.2.3/js/buttons.print.min.js"></script>
	<script type="text/javascript">

		$(document).ready(function() {
			$.noConflict();
			$('#mytable').DataTable( {
				dom: 'Bfrtip', 
				buttons: [
					'copy', 'csv', 'excel', 'pdf', 'print'
				]
				
			} );
		} );
   </script>

<body>
    <div>
	  <table id="mytable" class="table table-striped table-bordered" cellspacing="0" width="100%">
  <thead>
  <tr>
    <th>Ribozyme name</th>
    <th>Description </th>
    <th>Corresponding author</th>
    <th>Time</th>
    <th>Journal</th>
    <th>Pubmed ID</th>
  </tr>
 </thead>
<tbody>
  <tr>
    <td>Twister ribozyme</td>
    <td>Established an outstandingly flexible expression platform, which in conjugation with three different aptamer domains, enables the construction of many different one- and two-input regulators of gene expression in both bacteria and yeast.</td>
    <td>Jörg S. Hartig</td>
    <td>2016</td>
    <td>Nat Commun</td>
    <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/27670347" target="_blank"><b>27670347</b></a></td>
  </tr>
  <tr>
    <td>Twister ribozyme</td>
    <td>Established the Tornado (‘Twister-optimized RNA for durable overexpression’) expression system for rapid RNA circularization, resulting in RNA aptamers with high stability and expression levels.</td>
    <td>Samie R. Jaffrey</td>
    <td>2019</td>
    <td> Nat Biotechnol</td>
    <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/30962542" target="_blank"><b>30962542</b></a></td>
  </tr>
  <tr>
    <td>Twister ribozyme</td>
    <td>Established a photocaged nucleotide that releases guanosine within microseconds upon photosolvolysis with blue light. Application of this tool to O. sativa twister achieved the spatial (75 µm) and temporal (≤30 ms) control required to resolve folding and self-cleavage events when combined with single-molecule fluorescence detection of the ribozyme folding pathway.</td>
    <td>Sarah A Woodson </td>
    <td>2020</td>
    <td>Proc Natl Acad Sci U S A</td>
    <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/32430319" target="_blank"><b>32430319</b></a></td>
  </tr>
  <tr>
    <td>Diverse ribozymes,include hammerhead ribozyme,<br />twister ribozyme,twister sister ribozyme,<br />pistol ribozyme,<br />hairpin ribozyme,<br />varkud satellite ribozyme</td>
    <td>Established an RNA integrator comprising a ribozyme and an unfolded form of the fluorogenic aptamer Broccoli, which enables detection of low-abundance targets in vitro and in living cells.</td>
    <td>Samie R Jaffrey</td>
    <td>2019</td>
    <td>Cell Chem Biol</td>
    <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/30773480" target="_blank"><b>30773480</b></a></td>
  </tr>
  <tr>
    <td>Circularly permuted variants of the pistol ribozyme</td>
    <td>Investigated circularly permuted variants of the pistol ribozyme class (CPP) as a synthetic ribozyme scaffold for mammalian riboswitch, which allows chemical regulation of gene expression in mammalian cells.</td>
    <td>Yohei Yokobayashi</td>
    <td>2021</td>
    <td>ACS Synth Biol</td>
    <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/34374523" target="_blank"><b>34374523</b></a></td>
  </tr>
  <tr>
    <td>Hairpin ribozyme</td>
    <td>Engineering a number of hairpin ribozyme variants by programmed conformational design that carry out cascades of cleavage and ligation reactions, and as a result mediate more complex RNA processing reactions.</td>
    <td>Sabine Müller </td>
    <td>2019</td>
    <td>Ann N Y Acad Sci</td>
    <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/30941784" target="_blank"><b>30941784</b></a></td>
  </tr>
  <tr>
    <td>Hairpin ribozyme</td>
    <td>Situ activation of RNA substrates under reaction conditions amenable to catalysis by the hairpin ribozyme. Found that diamidophosphate (DAP) and imidazole drive the formation of 2',3'-cyclic phosphate RNA mono- and oligonucleotides from monophosphorylated precursors in frozen water-ice. This long-lived activation enables iterative enzymatic assembly of long RNAs.</td>
    <td>Hannes Mutschler </td>
    <td>2021</td>
    <td>Angew Chem Int Ed Engl</td>
    <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/33128282" target="_blank"><b>33128282</b></a></td>
  </tr>
  <tr>
    <td>Hairpin ribozyme</td>
    <td>Demonstrated the activity of engineered hairpin ribozyme variants, which as self-splicing introns are removed from their parent RNA.</td>
    <td>Sabine Müller</td>
    <td>2022</td>
    <td>Nucleic Acids Res</td>
    <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/34928378" target="_blank"><b>34928378</b></a></td>
  </tr>
  <tr>
    <td>GlmS ribozyme</td>
    <td>In addition to being a superb experimental system with which to analyze how RNA catalysts can exploit small molecule coenzymes to broaden their chemical versatility, the presence of the glmS ribozyme in numerous pathogenic bacteria make this RNA an attractive target for the development of new antibiotics and antibacterial strategies.</td>
    <td>Adrian R Ferré-D'Amaré</td>
    <td>2010</td>
    <td>Q Rev Biophys</td>
    <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/27670347" target="_blank"><b>27670355</b></a></td>
  </tr>
  <tr>
    <td>GlmS ribozyme</td>
    <td>Glucosamine-induced ribozyme activation can be rapidly reversed by removing the inducer.The glmS ribozyme could be used as a tool to study essential genes in T. brucei.</td>
    <td>Roberto Docampo</td>
    <td>2018 </td>
    <td>J Eukaryot Microbiol</td>
    <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/29265590" target="_blank"><b>29265590</b></a></td>
  </tr>
  <tr>
    <td>GlmS ribozyme</td>
    <td>The glmS riboswitch targeting the antisense oligonucleotide is sufficient to inhibit the growth of S. aureus with a MIC80 of 5 μg/mL. The glmS ribozyme is a very suitable target for antibacterial drug development with antisense oligonucleotides.</td>
    <td>Robert Penchovsky </td>
    <td>2021</td>
    <td>ACS Synth Biol</td>
    <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/27670347" target="_blank"><b>27670357</b></a></td>
  </tr>
  <tr>
    <td>HDV ribozyme</td>
    <td>Use of Hepatitis delta virus (HDV) ribozyme allows preparation of homogeneous RNA with any 3′-terminal sequence.</td>
    <td>J A Doudna</td>
    <td>1996</td>
    <td>Nucleic Acids Res</td>
    <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/8600468" target="_blank"><b>8600468</b></a></td>
  </tr>
  <tr>
    <td>HDV ribozyme</td>
    <td>Designed a small molecule responsive allosteric ribozymes based on the genomic hepatitis delta virus (HDV) ribozyme by replacing the P4-L4 stem-loop with an RNA aptamer through a connector stem，which can regulate  cis-gene expression by theophylline and guanine by up to 29.5-fold in mammalian cell culture.</td>
    <td>Yohei Yokobayashi</td>
    <td>2013</td>
    <td>ACS Synthetic Biology</td>
    <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/23697539" target="_blank"><b>23697539</b></a></td>
  </tr>
  <tr>
    <td>HDV ribozyme</td>
    <td>Generating in vitro transcripts with homogenous 3' ends using HDV ribozyme.</td>
    <td>Ciesiolka, J</td>
    <td>2004</td>
    <td>Nucleic Acids Res</td>
    <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/14973333" target="_blank"><b>14973333</b></a></td>
  </tr>
  <tr>
    <td>HDV ribozyme</td>
    <td>Designed a RNAi-regulatory system  integrating a theophylline aptamer-fused Hepatitis delta virus (HDV) ribozyme with a pri-miRNA analogue to control gene expression mediated by RNAi.The system can control gene expression by adding theophylline into the culture media in dose dependent fashion.</td>
    <td>Renjun Pei</td>
    <td>2017</td>
    <td>Integrative Biology</td>
    <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/28548675" target="_blank"><b>28548675</b></a></td>
  </tr>
  <tr>
    <td>HDV ribozyme</td>
    <td>Designed an alternatively processed short hairpin RNA (shRNA) molecule termed AgoshRNA with a ~18 base pairs (bp) stem and a 3-5 nucleotides (nt) loop,included the hepatitis delta virus (HDV) ribozyme at the 3' terminus,which can enhance the silencing activity of the AgoshRNA molecules.</td>
    <td> Elena Herrera-Carrillo</td>
    <td>2021</td>
    <td>Methods in Molecular Biology</td>
    <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/32712923" target="_blank"><b>32712923</b></a></td>
  </tr>
  <tr>
    <td>HDV ribozyme</td>
    <td>Designed an improved CRISPR-Cas12a system included the Hepatitis delta virus (HDV) ribozyme  to facilitate accurate 3'-end processing of the crRNA transcript to produce precise molecules,which enhanced not only the gene editing efficiency, but also the activity of the catalytically inactive Cas12a-based CRISPR gene activation platform.</td>
    <td>Elena Herrera-Carrillo</td>
    <td>2021</td>
    <td>Methods in Molecular Biology</td>
    <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/32712922" target="_blank"><b>32712922</b></a></td>
  </tr>
  <tr>
    <td>VS ribozyme</td>
    <td>Describes an innovative approach for RNA engineering and establishes that the KLI of the trans VS ribozyme can be adapted to cleave other folded RNA substrates.</td>
    <td>Pascale Legault</td>
    <td>2016</td>
    <td>Nucleic Acids Res</td>
    <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/27166370" target="_blank"><b>27166370</b></a></td>
  </tr>
  <tr>
    <td>LC ribozyme</td>
    <td>Lariat capping ribozyme functions in vivo in the budding yeast Saccharomyces cerevisiae presumably without cofactors and that lariat capping occurs cotranscriptionally.</td>
    <td>Henrik Nielsen</td>
    <td>2017</td>
    <td>RNA</td>
    <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/28159804" target="_blank"><b>28159804</b></a></td>
  </tr>
  <tr>
    <td>LC ribozyme</td>
    <td>Lariat capping is thus a versatile tool for studying the importance of the 5' end structure of RNA molecules.</td>
    <td>Henrik Nielsen</td>
    <td>2021</td>
    <td>Methods Mol Biol</td>
    <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/32712925" target="_blank"><b>32712925</b></a></td>
  </tr>
  <tr>
    <td>Group II intron</td>
    <td>Implementation of targetrons in eukaryote provides the practical basis for potential applications of targeted group II introns in genetic engineering, functional genomics, and gene therapy.</td>
    <td>A M Lambowitz</td>
    <td>2000</td>
    <td>Science</td>
    <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/10903206" target="_blank"><b>10903206</b></a></td>
  </tr>
  <tr>
    <td>Group II intron</td>
    <td>As controllable gene targeting vectors for genetic manipulation of bacteria.</td>
    <td> A M Lambowitz</td>
    <td>2001</td>
    <td>Nat Biotechnol</td>
    <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/11731786" target="_blank"><b>11731786</b></a></td>
  </tr>
  <tr>
    <td>Group II intron</td>
    <td>Mobile group II introns have been used to develop a novel class of gene targeting vectors, targetrons, which employ base pairing for DNA target recognition and can thus be programmed to insert into any desired target DNA.</td>
    <td>Alan M Lambowitz</td>
    <td>2003</td>
    <td>Nucleic Acids Res</td>
    <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/12626707" target="_blank"><b>12626707</b></a></td>
  </tr>
  <tr>
    <td>Group II intron</td>
    <td>a universal gene knock-out system for the genus Clostridium.</td>
    <td>Nigel P Minton</td>
    <td>2007</td>
    <td>J Microbiol Methods</td>
    <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/17658189" target="_blank"><b>17658189</b></a></td>
  </tr>
  <tr>
    <td>Group II intron</td>
    <td>New genetic engineering tools, such as the group II intron has shown promise for genetic manipulation of bacteria and forecast the dawn of a new era for a tumour-targeted bacterial vector system for gene therapy of solid tumours.</td>
    <td>Jozef Anné</td>
    <td>2008</td>
    <td>Cancer Lett</td>
    <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/18063294" target="_blank"><b>18063294</b></a></td>
  </tr>
  <tr>
    <td>Group II intron</td>
    <td>the diversity of mobile group II introns can be exploited to provide a large variety of different target specificities and potentially other useful properties for gene targeting.</td>
    <td>Alan M Lambowitz</td>
    <td>2009</td>
    <td>RNA</td>
    <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/19155322" target="_blank"><b>19155322</b></a></td>
  </tr>
  <tr>
    <td>Group II intron</td>
    <td>Targetrons have found wide application in microbiological research, enabling gene targeting and genetic engineering of bacteria that had been intractable to other methods. </td>
    <td> Alan M Lambowitz</td>
    <td>2014</td>
    <td>Mob DNA</td>
    <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/24410776" target="_blank"><b>24410776</b></a></td>
  </tr>
  <tr>
    <td>Group II intron</td>
    <td>Because of their ability to recognize and modify target DNA sites with high efficiency, Group II intron pioneered the use of RNA guided site specific genome modification.</td>
    <td>Emmanuelle Charpentier</td>
    <td>2014</td>
    <td>Science</td>
    <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/25430774" target="_blank"><b>25430774</b></a></td>
  </tr>
  <tr>
    <td>Group II intron</td>
    <td>For gene targeting with the aid of computer algorithms.</td>
    <td>Nicolás Toro </td>
    <td>2014</td>
    <td>RNA Biol</td>
    <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/24646865" target="_blank"><b>24646865</b></a></td>
  </tr>
  <tr>
    <td>Group II intron</td>
    <td>RT domains are particularly suitable for creation of RNA seq libraries.</td>
    <td>Charles A Thornton</td>
    <td>2018</td>
    <td>Nucleic Acids Res</td>
    <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/29036654" target="_blank"><b>29036654</b></a></td>
  </tr>
  <tr>
    <td>RNase P</td>
    <td>Showed that the same riboswitch constructs are useful tools to investigate the impact of the 5′-leader structure on tRNA processing by different types of RNase P from A. thaliana and H. sapiens. Designed riboswitches can regulate not only bacterial RNase P-mediated tRNA processing, but also affect the maturation catalyzed by the corresponding eukaryotic enzymes, by the RNA/protein assemblies or protein-only versions. </td>
    <td>Mario Mörl</td>
    <td>2022</td>
    <td>RNA</td>
    <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/35022261" target="_blank"><b>35022261</b></a></td>
  </tr>
  <tr>
    <td>RNase P</td>
    <td>Presented and discuss some computational and biochemical methods, which were applied, on the one hand, to design riboswitch candidates de novo and, on the other hand, to monitor the switching activity and for the investigation of the postulated switching mechanism.</td>
    <td> Sven Findeiß</td>
    <td>2022</td>
    <td>Methods Mol Biol</td>
    <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/35666446" target="_blank"><b>35666446</b></a></td>
  </tr>
  <tr>
    <td>RNase P</td>
    <td>Reported a biotechnology strategy based on the re-localization of plant protein-only RNase P to the cytosol (CytoRP) to target plant viruses tRNA-like structures and thus hamper virus replication. The results show the potential of using CytoRP for combating some plant viral diseases.</td>
    <td>Philippe Giegé</td>
    <td>2021</td>
    <td>Nat Commun </td>
    <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/33579946" target="_blank"><b>33579946</b></a></td>
  </tr>
  <tr>
    <td>RNase P</td>
    <td>Showed that the same riboswitch constructs are useful tools to investigate the impact of the 5′-leader structure on tRNA processing by different types of RNase P from A. thaliana and H. sapiens. Designed riboswitches can regulate not only bacterial RNase P-mediated tRNA processing, but also affect the maturation catalyzed by the corresponding eukaryotic enzymes, by the RNA/protein assemblies or protein-only versions. </td>
    <td>Mario Mörl</td>
    <td>2022</td>
    <td>RNA</td>
    <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/35022261" target="_blank"><b>35022261</b></a></td>
  </tr>
  <tr>
    <td>RNase P</td>
    <td>Designed theophylline-dependent riboswitches regulating the processing of a specific precursor tRNA by directly controlling the accessibility for RNase P. The leader region is masked by the sequestor hairpin, and ligand binding induces a structural rearrangement that disrupts this hairpin, giving RNase P access to its cleavage position. Further, this design allows investigation of the impact of the tRNA 5′-leader structure on RNase P mediated processing.</td>
    <td>Mario Mörl</td>
    <td>2021</td>
    <td>Nucleic Acids Res</td>
    <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/33469651" target="_blank"><b>33469651</b></a></td>
  </tr>
  <tr>
    <td>RNase P</td>
    <td>The RNase P-based M1GS ribozyme is a promising antisense technique in HCV therapeutic studies. The M1GS ribozyme can function as an antiviral agent and effectively inhibit gene expression and multiplication of HCV.</td>
    <td>Gang Li</td>
    <td>2014</td>
    <td>Virology journal </td>
    <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/24885776" target="_blank"><b>24885776</b></a></td>
  </tr>
  <tr>
    <td>RNase P</td>
    <td>A sequence-specific ribozyme (M1GS RNA) derived from the catalytic RNA subunit of RNase P from Escherichia coli was used to target the overlapping exon 3 region of the mRNAs encoding the major transcription regulatory proteins IE1 and IE2 of human cytomegalovirus.  This study is the first to use M1GS ribozyme for inhibiting human cytomegalovirus replication and demonstrates the utility of this ribozyme for antiviral applications.</td>
    <td>Fenyong Liu</td>
    <td>2000</td>
    <td>Proc Natl Acad Sci U S A</td>
    <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/10811889" target="_blank"><b>10811889</b></a></td>
  </tr>
  <tr>
    <td>Hammerhead ribozyme</td>
    <td>Incorporating a hammerhead ribozyme on the 5′-end of d56 together with the VSsl at the3′-terminus to overcome 5′ heterogeneity.</td>
    <td>Jennifer A. Doudna</td>
    <td>1996</td>
    <td>Nucleic Acids Res<br /> &nbsp;&nbsp;</td>
    <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/8600468" target="_blank"><b>8600468</b></a></td>
  </tr>
  <tr>
    <td>Hammerhead ribozyme</td>
    <td>The catalytic activity of the hammerhead ribozyme can be brought under the control of ATP and theophylline by designing conjoined aptamer-ribozyme complexes.</td>
    <td>R R Breaker</td>
    <td>1997</td>
    <td>Chem Biol</td>
    <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/9224568" target="_blank"><b>9224568</b></a></td>
  </tr>
  <tr>
    <td>Hammerhead ribozyme</td>
    <td>The hammerhead ribozyme is established RNA switches as designer molecular sensors of FMN.</td>
    <td>R R Breaker</td>
    <td>1999</td>
    <td>Proc Natl Acad Sci U S A<br /> &nbsp;&nbsp;</td>
    <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/10097080" target="_blank"><b>10097080</b></a></td>
  </tr>
  <tr>
    <td>Hammerhead ribozyme</td>
    <td>Heterogeneous-length RNA transcripts are processed with a hammerhead ribozyme to yield homogenous-length products.</td>
    <td>A Pardi</td>
    <td>1999</td>
    <td>RNA<br /> &nbsp;&nbsp;</td>
    <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/10496226" target="_blank"><b>10496226</b></a></td>
  </tr>
  <tr>
    <td>Hammerhead ribozyme</td>
    <td>The hammerhead ribozyme is used to interfere with HIV-1 gene expression.</td>
    <td>E Luzi, F Eckstein</td>
    <td>2000</td>
    <td>Nucleic Acids Res<br /> &nbsp;&nbsp;</td>
    <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/11058100" target="_blank"><b>11058100</b></a></td>
  </tr>
  <tr>
    <td>Hammerhead ribozyme</td>
    <td>The hammerhead ribozyme is integrated into the transcription unit of a gene or vector leading to potent inhibition of gene or vector expression.</td>
    <td>Richard C Mulligan</td>
    <td>2004</td>
    <td>Nature</td>
    <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/15386015" target="_blank"><b>15386015</b></a></td>
  </tr>
  <tr>
    <td>Hammerhead ribozyme</td>
    <td>The hammerhead ribozyme constructs an aptazyme with a theophylline aptamer and a communication module,whose conformational switches inhibit or promote ribosome binding for regulating downstream gene expression.</td>
    <td>Mizuo Maeda</td>
    <td>2008</td>
    <td>Chembiochem<br /> &nbsp;&nbsp;</td>
    <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/18098257" target="_blank"><b>18098257</b></a></td>
  </tr>
    <tr>
    <td>Group I intron</td>
    <td>many researchers have engineered self-splicing ribozymes to trans-splicing ones for repairing or reprograming disease-associated genes to corresponding wild-type or entirely new genes for therapeutic purposes</td>
    <td>Seong-Wook Lee</td>
    <td>2018</td>
    <td>Wiley Interdiscip Rev RNA</td>
    <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/29383855" target="_blank"><b>29383855</b></a></td>
  </tr>
  <tr>
    <td>Group I intron</td>
    <td>①As a drug target:inhibition of ribozymes present in some pathogens but not in humans.② Exploiting the catalytic functions of ribozyme to target other cellular RNA of our interest;act as therapeutics to correct the defective cellular RNA by trans-splicing activity and are renowned as equivalent as that of any gene therapy for genetic disorders or it can be a “gene inhibitor” as it can cleave the target RNA.</td>
    <td></td>
    <td>2010</td>
    <td>Recent Pat DNA Gene Seq</td>
    <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/20218956" target="_blank"><b>20218956</b></a></td>
  </tr>
  <tr>
    <td>Group I intron</td>
    <td>Expressing circular RNA of desired sequence to Study the function of circular RNA and RNA topology in vivo.</td>
    <td> M Ares Jr</td>
    <td>1994</td>
    <td>Proc Natl Acad Sci U S A</td>
    <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/7512723" target="_blank"><b>7512723</b></a></td>
  </tr>
  <tr>
    <td>Group I intron</td>
    <td>Intron fusion represents a novel method of transcript stabilization, which is of potential use to augment other means of increasing gene expression for purposes of product amplification.</td>
    <td>G Belfort, M Belfort</td>
    <td>1988</td>
    <td>Gene</td>
    <td><a href="https://www.ncbi.nlm.nih.gov/pubmed/2468580" target="_blank"><b>2468580</b></a></td>
  </tr>
</tbody>
</table><br /><br />



</div></body></html>

</div>

      </div>
    </div>

    <div id="footer" class="panel">
  <div class="panel-footer" style='background-color: #333333; color: #eeeeee;'>
	<div class="container-fluid">
	  <div class="row">
		<div class="col-sm-4">
			
		  <p>&copy 2024 GZNL-RDC. </p>
		  <p>We are part of the <a href="https://gzlab.ac.cn">Guangzhou Laboratory</a> and <a href="http://www.rnacentre.org/">RNAcentre</a>.</p>
		  <p>Site made with <a href="https://jekyllrb.com">Jekyll</a>;  <a href="http://localhost:4000/aboutwebsite.html">copy and  modify it for your own research group.</a></p>
		  <p>  </p>
		  <p>  </p>
		</div>
		
		<div class="col-sm-4">
		  Funding:<br />
          - <a href="https://gzlab.ac.cn">R&D Programs of Guangzhou Laboratory</a><br />
          - <a href="http://www.most.gov.cn/">MOST IT+BT projects</a> <br />
          - <a href="https://www.nsfc.gov.cn/">NSFC</a> <br />	  
		</div>
		
		<div class="col-sm-4">
		  Contact:<br />
		  Building B2, Guangzhou National Laboratory<br />
		  96 Xingdao South Road, Guangzhou International Bio Island, <br />
		  Haizhu District, Guangzhou, Guangdong, China.<br />
          (<a href="https://goo.gl/maps/hDLr6siEAS2Eaypt8">Maps</a>)
		</div>
	  </div>
	</div>
  </div>
</div>

<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
<script src="http://localhost:4000/js/bootstrap.min.js"></script>


  </body>

</html>
