<tr>
    <td >Hammerhead</td>
    <td >Three dimensional structure of a hammerhead ribozyme</td>
    <td >N/A</td>
    <td >N/A</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/1HMH"><b> 1HMH</b></a> </td>
    <td >2.6</td>
    <td >1994</td>
    <td >a</td>
  </tr>
  <tr>
    <td >Hammerhead</td>
    <td >The 2.2 A structure of a full-length catalytically active hammerhead ribozyme</td>
    <td >Schistosoma mansoni</td>
    <td >N/A</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/3ZD5"><b> 3ZD5</b></a> </td>
    <td >2.2</td>
    <td >2006</td>
    <td >Cell</td>
  </tr>
  <tr>
    <td >Hammerhead</td>
    <td >Hammerhead Ribozyme G12A mutant pre-cleavage</td>
    <td >N/A</td>
    <td >N/A</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/2QUS"><b> 2QUS</b></a> </td>
    <td >2.4</td>
    <td >2008</td>
    <td >PLoS Biol</td>
  </tr>
  <tr>
    <td >Hammerhead</td>
    <td >High-resolution full-length hammerhead ribozyme</td>
    <td >synthetic construct</td>
    <td >N/A</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/3ZP8"><b> 3ZP8</b></a> </td>
    <td >1.55</td>
    <td >2013</td>
    <td >J Mol Biol</td>
  </tr>
  <tr>
    <td >Hammerhead</td>
    <td >the crystal structure of an all-rna hammerhead ribozyme: a proposed mechanism for rna catalytic cleavage</td>
    <td >N/A</td>
    <td >N/A</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/1MME"><b> 1MME</b></a> </td>
    <td >3.1</td>
    <td >1995</td>
    <td >Cell</td>
  </tr>
  <tr>
    <td >Hammerhead</td>
    <td >Full-Length Hammerhead Ribozyme with G12A substitution at the general base position</td>
    <td >synthetic construct</td>
    <td >N/A</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/3ZD4"><b> 3ZD4</b></a> </td>
    <td >2.2</td>
    <td >2014</td>
    <td >Acta Crystallogr D Biol Crystallogr</td>
  </tr>
  <tr>
    <td >Hammerhead</td>
    <td >inhibition of the hammerhead ribozyme cleavage reaction by site-specific binding of tb(iii)</td>
    <td >N/A</td>
    <td >N/A</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/359D"><b> 359D</b></a> </td>
    <td >2.9</td>
    <td >1998</td>
    <td >Science</td>
  </tr>
  <tr>
    <td >Hammerhead</td>
    <td >capturing the structure of a catalytic rna intermediate: rna hammerhead ribozyme, mg(ii)-soaked</td>
    <td >N/A</td>
    <td >N/A</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/301D"><b> 301D</b></a> </td>
    <td >3.0 </td>
    <td >1996</td>
    <td >Science</td>
  </tr>
  <tr>
    <td >Hammerhead</td>
    <td >capturing the structure of a catalytic rna intermediate: the hammerhead ribozyme</td>
    <td >N/A</td>
    <td >N/A</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/299D"><b> 299D</b></a> </td>
    <td >3</td>
    <td >1996</td>
    <td >Science</td>
  </tr>
  <tr>
    <td >Hammerhead</td>
    <td >Hammerhead Ribozyme G12A mutant after cleavage</td>
    <td >N/A</td>
    <td >N/A</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/2QUW"><b> 2QUW</b></a> </td>
    <td >2.2</td>
    <td >2008</td>
    <td >PLoS Biol</td>
  </tr>
  <tr>
    <td >Hammerhead</td>
    <td >The structural basis of hammerhead ribozyme self-cleavage</td>
    <td >N/A</td>
    <td >N/A</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/379D"><b> 379D</b></a> </td>
    <td >3.1</td>
    <td >1998</td>
    <td >Cell</td>
  </tr>
  <tr>
    <td >Hammerhead</td>
    <td >Hammerhead Ribozyme with 5'-5' G-G linkage: Conformational change experiment</td>
    <td >N/A</td>
    <td >N/A</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/1Q29"><b> 1Q29</b></a> </td>
    <td >3</td>
    <td >2003</td>
    <td >J Mol Biol</td>
  </tr>
  <tr>
    <td >Hammerhead</td>
    <td >Two divalent metal ions and conformational changes play roles in the hammerhead ribozyme cleavage reaction-G12A mutant in Mg2+</td>
    <td >synthetic construct</td>
    <td >N/A</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/5DH6"><b> 5DH6</b></a> </td>
    <td >2.78</td>
    <td >2015</td>
    <td >Biochemistry</td>
  </tr>
  <tr>
    <td >Hammerhead</td>
    <td >a three-dimensional model for the hammerhead ribozyme based on fluorescence measurements</td>
    <td >N/A</td>
    <td >N/A</td>
    <td >Fluorescence transfer</td>
    <td ><a href="https://www.rcsb.org/structure/1RMN"><b> 1RMN</b></a> </td>
    <td >N/A</td>
    <td >1994</td>
    <td >Science</td>
  </tr>
  <tr>
    <td >Hammerhead</td>
    <td >capturing the structure of a catalytic rna intermediate: rna hammerhead ribozyme, mn(ii)-soaked</td>
    <td >N/A</td>
    <td >N/A</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/300D"><b> 300D</b></a> </td>
    <td >3</td>
    <td >1996</td>
    <td >Science</td>
  </tr>
  <tr>
    <td >Hammerhead</td>
    <td >Crosslinked Hammerhead Ribozyme Initial State</td>
    <td >N/A</td>
    <td >N/A</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/1NYI"><b> 1NYI</b></a> </td>
    <td >2.85</td>
    <td >2003</td>
    <td >J Mol Biol</td>
  </tr>
  <tr>
    <td >Hammerhead</td>
    <td >Two divalent metal ions and conformational changes play roles in the hammerhead ribozyme cleavage reaction-WT ribozyme in Mg2+</td>
    <td >synthetic construct</td>
    <td >N/A</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/5DQK"><b> 5DQK</b></a> </td>
    <td >2.71</td>
    <td >2015</td>
    <td >Biochemistry</td>
  </tr>
  <tr>
    <td >Hammerhead</td>
    <td >Full-length hammerhead ribozyme with Mn(II) bound</td>
    <td >N/A</td>
    <td >N/A</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/2OEU"><b> 2OEU</b></a> </td>
    <td >2</td>
    <td >2008</td>
    <td >Chem Biol</td>
  </tr>
  <tr>
    <td >Hammerhead</td>
    <td >Two divalent metal ions and conformational changes play roles in the hammerhead ribozyme cleavage reaction- G12A mutant in Zn2+</td>
    <td >synthetic construct</td>
    <td >N/A</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/5DH8"><b> 5DH8</b></a> </td>
    <td >3.3</td>
    <td >2015</td>
    <td >Biochemistry</td>
  </tr>
  <tr>
    <td >Hammerhead</td>
    <td >Two active site divalent ion in the crystal structure of the hammerhead ribozyme bound to a transition state analog-Mg2+</td>
    <td >synthetic construct</td>
    <td >N/A</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/5EAO"><b> 5EAO</b></a> </td>
    <td >2.99</td>
    <td >2016</td>
    <td >Biochemistry</td>
  </tr>
  <tr>
    <td >Hammerhead</td>
    <td >Two divalent metal ions and conformational changes play roles in the hammerhead ribozyme cleavage reaction-WT ribozyme in Mn2+ at low pH</td>
    <td >synthetic construct</td>
    <td >N/A</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/5DI4"><b> 5DI4</b></a> </td>
    <td >2.95</td>
    <td >2015</td>
    <td >Biochemistry</td>
  </tr>
  <tr>
    <td >Hammerhead</td>
    <td >Two divalent metal ions and conformational changes play roles in the hammerhead ribozyme cleavage reaction-WT ribozyme in Mn2+ at high pH</td>
    <td >synthetic construct</td>
    <td >N/A</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/5DI2<br> &nbsp;&nbsp;"><b> 5DI2<br> &nbsp;&nbsp;</b></a> </td>
    <td >2.99</td>
    <td >2015</td>
    <td >Biochemistry</td>
  </tr>
  <tr>
    <td >Hammerhead</td>
    <td >Two divalent metal ions and conformational changes play roles in the hammerhead ribozyme cleavage reaction-G12A mutant in Mn2+</td>
    <td >synthetic construct</td>
    <td >N/A</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/5DH7"><b> 5DH7</b></a> </td>
    <td >3.06</td>
    <td >2015</td>
    <td >Biochemistry</td>
  </tr>
  <tr>
    <td >Hammerhead</td>
    <td >Two active site divalent ion in the crystal structure of the hammerhead ribozyme bound to a transition state analog-Mn2+</td>
    <td >synthetic construct</td>
    <td >N/A</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/5EAQ"><b> 5EAQ</b></a> </td>
    <td >3.2</td>
    <td >2016</td>
    <td >Biochemistry</td>
  </tr>
  <tr>
    <td >HDV </td>
    <td >The structure of the isolated, central hairpin of the hdv antigenomic ribozyme, nmr, 10 structures</td>
    <td >Hepatitis delta virus</td>
    <td >N/A</td>
    <td >SOLUTION NMR</td>
    <td ><a href="https://www.rcsb.org/structure/1ATO"><b> 1ATO</b></a> </td>
    <td >N/A</td>
    <td >1997</td>
    <td >EMBO J</td>
  </tr>
  <tr>
    <td >HDV </td>
    <td >U1a spliceosomal protein/hepatitis delta virus genomic ribozyme complex </td>
    <td >Hepatitis delta virus, Homo sapiens</td>
    <td >MAD</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/1DRZ"><b> 1DRZ</b></a> </td>
    <td >2.3</td>
    <td >1998</td>
    <td >Nature</td>
  </tr>
  <tr>
    <td >HDV </td>
    <td >Crystal structure of a C75U mutant Hepatitis Delta Virus ribozyme precursor, in Cu2+ solution</td>
    <td >Hepatitis delta virus, Homo sapiens</td>
    <td >Se-MAD,Ir-SAD,MR</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/1SJ4"><b> 1SJ4</b></a> </td>
    <td >2.7</td>
    <td >2004</td>
    <td >Nature</td>
  </tr>
  <tr>
    <td >HDV </td>
    <td >Hepatitis Delta Virus Gemonic Ribozyme Precursor, with Mg2+ Bound</td>
    <td >Hepatitis delta virus, Homo sapiens</td>
    <td >Se-MAD,Ir-SAD,MR</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/1SJ3"><b> 1SJ3</b></a> </td>
    <td >2.2</td>
    <td >2004</td>
    <td >Nature</td>
  </tr>
  <tr>
    <td >HDV </td>
    <td >Crystal Structure of the Hepatitis Delta Virus Gemonic Ribozyme Product with C75U Mutaion, cleaved in Imidazole and Mg2+ solutions</td>
    <td >Hepatitis delta virus, Homo sapiens</td>
    <td >Se-MAD,Ir-SAD,MR</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/1VC6"><b> 1VC6</b></a> </td>
    <td >2.8</td>
    <td >2004</td>
    <td >Nature</td>
  </tr>
  <tr>
    <td >HDV </td>
    <td >Crystal Structure of the Wild Type Hepatitis Delta Virus Gemonic Ribozyme Precursor, in EDTA solution</td>
    <td >Homo sapiens</td>
    <td >Se-MAD,Ir-SAD,MR</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/1VC5"><b> 1VC5</b></a> </td>
    <td >3.4</td>
    <td >2004</td>
    <td >Nature</td>
  </tr>
  <tr>
    <td >HDV </td>
    <td >Crystal Structure of the Hepatitis Delta Virus Gemonic Ribozyme Precursor, with C75U mutaion, in Imidazole and Sr2+ solution</td>
    <td >Homo sapiens</td>
    <td >Se-MAD,Ir-SAD,MR</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/1VC0"><b> 1VC0</b></a> </td>
    <td >2.5</td>
    <td >2004</td>
    <td >Nature</td>
  </tr>
  <tr>
    <td >HDV </td>
    <td >Crystal Structure of the Hepatitis Delta Virus Gemonic Ribozyme Precursor, with C75U mutaion, in Ba2+ solution</td>
    <td >Homo sapiens</td>
    <td >Se-MAD,Ir-SAD,MR</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/1VBZ"><b> 1VBZ</b></a> </td>
    <td >2.8</td>
    <td >2004</td>
    <td >Nature</td>
  </tr>
  <tr>
    <td >HDV </td>
    <td >Crystal Structure of the Hepatitis Delta Virus Gemonic Ribozyme Precursor, with C75U mutaion, in EDTA solution</td>
    <td >Homo sapiens</td>
    <td >Se-MAD,Ir-SAD,MR</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/1VBX"><b> 1VBX</b></a> </td>
    <td >2.7</td>
    <td >2004</td>
    <td >Nature</td>
  </tr>
  <tr>
    <td >HDV </td>
    <td >Crystal Structure of the Hepatitis Delta Virus Gemonic Ribozyme Precursor, with C75U mutaion, in Cobalt Hexammine solution</td>
    <td >Homo sapiens</td>
    <td >Se-MAD,Ir-SAD,MR</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/1SJF"><b> 1SJF</b></a> </td>
    <td >2.75</td>
    <td >2004</td>
    <td >Nature</td>
  </tr>
  <tr>
    <td >HDV </td>
    <td >Crystal Structure of the Hepatitis Delta Virus Gemonic Ribozyme Precursor, with C75U mutaion, and Mn2+ bound</td>
    <td >Homo sapiens</td>
    <td >Se-MAD,Ir-SAD,MR</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/1VBY"><b> 1VBY</b></a> </td>
    <td >2.9</td>
    <td >2004</td>
    <td >Nature</td>
  </tr>
  <tr>
    <td >HDV </td>
    <td >Hepatitis Delta Virus ribozyme precursor structure, with C75U mutation, bound to Tl+ and cobalt hexammine (Co(NH3)63+)</td>
    <td >Homo sapiens</td>
    <td >MR (Tl+)</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/2OJ3"><b> 2OJ3</b></a> </td>
    <td >2.9</td>
    <td >2007</td>
    <td >Structure</td>
  </tr>
  <tr>
    <td >HDV </td>
    <td >Hepatitis Delta Virus gemonic ribozyme precursor with C75U mutation and bound to monovalent cation Tl+</td>
    <td >Homo sapiens</td>
    <td >MR (Tl+)</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/2OIH"><b> 2OIH</b></a> </td>
    <td >2.4</td>
    <td >2007</td>
    <td >Structure</td>
  </tr>
  <tr>
    <td >HDV </td>
    <td >A 1.9A crystal structure of the HDV ribozyme precleavage suggests both Lewis acid and general acid mechanisms contribute to phosphodiester cleavage</td>
    <td >Homo sapiens</td>
    <td >MR</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/3NKB"><b> 3NKB</b></a> </td>
    <td >1.92</td>
    <td >2010</td>
    <td >Biochemistry</td>
  </tr>
  <tr>
    <td >HDV </td>
    <td >A Second Look at the HDV Ribozyme Structure and Dynamics.</td>
    <td >Homo sapiens</td>
    <td >Rebuild</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/4PR6(原1CX0)"><b> 4PR6(原1CX0)</b></a> </td>
    <td >2.3</td>
    <td >2014</td>
    <td >Nucleic Acids Res</td>
  </tr>
  <tr>
    <td >HDV </td>
    <td >Hepatitis delta virus ribozyme</td>
    <td >Hepatitis delta virus, Homo sapiens</td>
    <td >MAD</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/1CX0"><b> 1CX0</b></a> </td>
    <td >N/A</td>
    <td >1998</td>
    <td >Nature</td>
  </tr>
  <tr>
    <td >HDV </td>
    <td >A Second Look at the HDV Ribozyme Structure and Dynamics.</td>
    <td >Homo sapiens</td>
    <td >Rebuild</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/4PRF(原1VC7)"><b> 4PRF(原1VC7)</b></a> </td>
    <td >2.4</td>
    <td >2014</td>
    <td >Nucleic Acids Res</td>
  </tr>
  <tr>
    <td >HDV </td>
    <td >Crystal Structure of the Hepatitis Delta Virus Gemonic Ribozyme Precursor, with C75U mutaion, in Sr2+ solution</td>
    <td >Hepatitis delta virus, Homo sapiens</td>
    <td >Se-MAD,Ir-SAD,MR</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/1VC7"><b> 1VC7</b></a> </td>
    <td >N/A</td>
    <td >2004</td>
    <td >Nature</td>
  </tr>
  <tr>
    <td >Hairpin </td>
    <td >Crystal structure of a hairpin ribozyme in the catalytically-active conformation</td>
    <td >tobacco ringspot virus</td>
    <td >MR</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/1M5K"><b> 1M5K</b></a> </td>
    <td >2.4</td>
    <td >2002</td>
    <td >Science</td>
  </tr>
  <tr>
    <td >Hairpin </td>
    <td >Crystal structure of a junctionless all-RNA hairpin ribozyme </td>
    <td >tobacco ringspot virus</td>
    <td >MR</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/2OUE"><b> 2OUE</b></a> </td>
    <td >2.05</td>
    <td >2006</td>
    <td >Biochemistry </td>
  </tr>
  <tr>
    <td >VS</td>
    <td >Crystal structure of the VS ribozyme - G638A mutant</td>
    <td >Neurospora</td>
    <td >SAD-Ir</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/4R4V"><b> 4R4V</b></a> </td>
    <td >3.1</td>
    <td >2015</td>
    <td >Nat Chem Biol </td>
  </tr>
  <tr>
    <td >VS</td>
    <td >Crystal Structure of the VS ribozyme-A756G mutant</td>
    <td >Neurospora</td>
    <td >MR</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/4R4P"><b> 4R4P</b></a> </td>
    <td >3.1</td>
    <td >2015</td>
    <td >Nat Chem Biol </td>
  </tr>
  <tr>
    <td >VS</td>
    <td >Crystal structure of the VS ribozyme - wild-type C634</td>
    <td >Neurospora</td>
    <td >MR</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/5V3I"><b> 5V3I</b></a> </td>
    <td >3.3</td>
    <td >2017</td>
    <td >J Am Chem Soc</td>
  </tr>
  <tr>
    <td >VS</td>
    <td >NMR structure of the A730 loop of the Neurospora VS ribozyme</td>
    <td >Neurospora</td>
    <td >N/A</td>
    <td >NMR</td>
    <td ><a href="https://www.rcsb.org/structure/2L5Z"><b> 2L5Z</b></a> </td>
    <td >N/A</td>
    <td >2011</td>
    <td >Nucleic Acids Res</td>
  </tr>
  <tr>
    <td >VS</td>
    <td >Solution Structure of the Active Site Stem-Loop of the VS Ribozyme</td>
    <td >N/A</td>
    <td >N/A</td>
    <td >NMR</td>
    <td ><a href="https://www.rcsb.org/structure/1TJZ"><b> 1TJZ</b></a> </td>
    <td >N/A</td>
    <td >2004</td>
    <td >J Mol Biol </td>
  </tr>
  <tr>
    <td >VS</td>
    <td >NMR Structure of the Active Conformation of the VS Ribozyme Cleavage Site</td>
    <td >N/A</td>
    <td >N/A</td>
    <td >NMR</td>
    <td ><a href="https://www.rcsb.org/structure/1OW9"><b> 1OW9</b></a> </td>
    <td >N/A</td>
    <td >2003</td>
    <td >PNAS</td>
  </tr>
  <tr>
    <td >VS</td>
    <td >NMR Localization of Divalent Cations at the Active Site of the Neurospora VS Ribozyme Provides Insights Into RNA-Metal Ion Interactions</td>
    <td >Neurospora</td>
    <td >N/A</td>
    <td >NMR</td>
    <td ><a href="https://www.rcsb.org/structure/2MIS"><b> 2MIS</b></a> </td>
    <td >N/A</td>
    <td >2014</td>
    <td >Biochemistry</td>
  </tr>
  <tr>
    <td >VS</td>
    <td >NMR structure of the II-III-VI three-way junction from the VS ribozyme</td>
    <td >synthetic construct</td>
    <td >N/A</td>
    <td >NMR</td>
    <td ><a href="https://www.rcsb.org/structure/2N3Q"><b> 2N3Q</b></a> </td>
    <td >N/A</td>
    <td >2015</td>
    <td >RNA</td>
  </tr>
  <tr>
    <td >VS</td>
    <td >SOLUTION STRUCTURE OF THE VS RIBOZYME SUBSTRATE STEM-LOOP</td>
    <td >N/A</td>
    <td >N/A</td>
    <td >NMR</td>
    <td ><a href="https://www.rcsb.org/structure/1HWQ"><b> 1HWQ</b></a> </td>
    <td >N/A</td>
    <td >2001</td>
    <td >J Mol Biol </td>
  </tr>
  <tr>
    <td >VS</td>
    <td >NMR structure of the II-III-VI three-way junction from the VS ribozyme and identification of magnesium-binding sites using paramagnetic relaxation enhancement</td>
    <td >Neurospora crassa</td>
    <td >N/A</td>
    <td >NMR</td>
    <td ><a href="https://www.rcsb.org/structure/2N3R"><b> 2N3R</b></a> </td>
    <td >N/A</td>
    <td >2015</td>
    <td >RNA</td>
  </tr>
  <tr>
    <td >VS</td>
    <td >Solution structure of the VS ribozyme stem-loop V in the presence of MgCl2</td>
    <td >N/A</td>
    <td >N/A</td>
    <td >NMR</td>
    <td ><a href="https://www.rcsb.org/structure/1YN1"><b> 1YN1</b></a> </td>
    <td >N/A</td>
    <td >2006</td>
    <td >Biochemistry</td>
  </tr>
  <tr>
    <td >VS</td>
    <td >Solution structure of the Neurospora VS ribozyme stem-loop V in the presence of MgCl2 with modeling of bound manganese ions</td>
    <td >N/A</td>
    <td >N/A</td>
    <td >NMR</td>
    <td ><a href="https://www.rcsb.org/structure/1YN2"><b> 1YN2</b></a> </td>
    <td >N/A</td>
    <td >2006</td>
    <td >Biochemistry</td>
  </tr>
  <tr>
    <td >VS</td>
    <td >NMR structure of the VS ribozyme stem-loop V RNA in the absence of multivalent ions.</td>
    <td >N/A</td>
    <td >N/A</td>
    <td >NMR</td>
    <td ><a href="https://www.rcsb.org/structure/1TBK"><b> 1TBK</b></a> </td>
    <td >N/A</td>
    <td >2005</td>
    <td >Biochemistry</td>
  </tr>
  <tr>
    <td >VS</td>
    <td >NMR structure of the III-IV-V three-way junction from the VS ribozyme</td>
    <td >Neurospora crassa</td>
    <td >N/A</td>
    <td >NMR</td>
    <td ><a href="https://www.rcsb.org/structure/2MTJ"><b> 2MTJ</b></a> </td>
    <td >N/A</td>
    <td >2014</td>
    <td >Biochemistry</td>
  </tr>
  <tr>
    <td >VS</td>
    <td >NMR structure of the III-IV-V three-way junction from the VS ribozyme and identification of magnesium-binding sites using paramagnetic relaxation enhancement</td>
    <td >Neurospora crassa</td>
    <td >N/A</td>
    <td >NMR</td>
    <td ><a href="https://www.rcsb.org/structure/2MTK"><b> 2MTK</b></a> </td>
    <td >N/A</td>
    <td >2014</td>
    <td >Biochemistry</td>
  </tr>
  <tr>
    <td >VS</td>
    <td >NMR structure of the I-V kissing-loop interaction of the Neurospora VS ribozyme</td>
    <td >Neurospora crassa</td>
    <td >N/A</td>
    <td >NMR</td>
    <td ><a href="https://www.rcsb.org/structure/2MI0"><b> 2MI0</b></a> </td>
    <td >N/A</td>
    <td >2014</td>
    <td >Biochemistry</td>
  </tr>
  <tr>
    <td >glmS </td>
    <td >Pre-cleavage state of glmS ribozyme bound to glucose-6-phosphate</td>
    <td >Thermoanaerobacter tengcongensis</td>
    <td >MIRAS</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/2HO7"><b> 2HO7</b></a> </td>
    <td >2.9</td>
    <td >2006</td>
    <td >Science</td>
  </tr>
  <tr>
    <td >glmS </td>
    <td >Post-cleavage state of glmS ribozyme</td>
    <td >Thermoanaerobacter tengcongensis </td>
    <td >MIRAS</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/2GCV"><b> 2GCV</b></a> </td>
    <td >2.1</td>
    <td >2006</td>
    <td >Science</td>
  </tr>
  <tr>
    <td >glmS </td>
    <td >Structural investigation of the GlmS ribozyme bound to its catalytic cofactor</td>
    <td >Bacillus anthracis</td>
    <td class="tg-0lax"> SIRAS</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/2NZ4"><b> 2NZ4</b></a> </td>
    <td >2.5</td>
    <td >2007</td>
    <td >Chemistry &amp; biology</td>
  </tr>
  <tr>
    <td >Twister</td>
    <td >Crystal Structure of the Twister Ribozyme with the Nucleotide 5'- to the Cleavage Site</td>
    <td >N/A</td>
    <td >MAD+SIRAS-Cs/Sm</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/4QJH"><b> 4QJH</b></a> </td>
    <td >3.88</td>
    <td >2014</td>
    <td >Proc Natl Acad Sci U S A</td>
  </tr>
  <tr>
    <td >Twister</td>
    <td >Crystal Structure of Twister with the Nucleotide 5'- to the Cleavage Site</td>
    <td >N/A</td>
    <td >MR+SIR-Ir/Co</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/4QJD"><b> 4QJD</b></a> </td>
    <td >3.1</td>
    <td >2014</td>
    <td >Proc Natl Acad Sci U S A</td>
  </tr>
  <tr>
    <td >Twister</td>
    <td >Crystal structure of the in-line aligned env22 twister ribozyme</td>
    <td >N/A</td>
    <td >Ir-SAD</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/4RGE"><b> 4RGE</b></a> </td>
    <td >2.89</td>
    <td >2014</td>
    <td >Nat Commun</td>
  </tr>
  <tr>
    <td >Twister</td>
    <td >Crystal structure of the in-line aligned env22 twister ribozyme soaked with Mn2+</td>
    <td >N/A</td>
    <td >Ir-SAD</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/4RGF"><b> 4RGF</b></a> </td>
    <td >3.2</td>
    <td >2014</td>
    <td >Nat Commun</td>
  </tr>
  <tr>
    <td >Twister</td>
    <td >The crystal structure of OMe substituted twister ribozyme</td>
    <td >N/A</td>
    <td >MR</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/5DUN"><b> 5DUN</b></a> </td>
    <td >2.64</td>
    <td >2015</td>
    <td >Angew Chem Int Ed Engl</td>
  </tr>
  <tr>
    <td >Twister</td>
    <td >Crystal Structure of Twister Ribozyme</td>
    <td >N/A</td>
    <td >Br-SAD</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/4OJI"><b> 4OJI</b></a> </td>
    <td >2.3</td>
    <td >2014</td>
    <td >Nat Chem Biol </td>
  </tr>
  <tr>
    <td >Twister-sister</td>
    <td >Structure-based Insights into Self-Cleavage by a Four-way Junctional Twister-Sister Ribozyme</td>
    <td >N/A</td>
    <td >Ir-SAD</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/5Y87"><b> 5Y87</b></a> </td>
    <td >2.13</td>
    <td >2017</td>
    <td >Nat Commun</td>
  </tr>
  <tr>
    <td >Twister-sister</td>
    <td >Structure-based Insights into Self-Cleavage by a Four-way Junctional Twister-Sister Ribozyme</td>
    <td >N/A</td>
    <td >Ir-SAD</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/5Y85"><b> 5Y85</b></a> </td>
    <td >2</td>
    <td >2017</td>
    <td >Nat Commun</td>
  </tr>
  <tr>
    <td >Twister-sister</td>
    <td >Crystal Structure of the Twister Sister (TS) Ribozyme at 2.0 Angstrom</td>
    <td >N/A</td>
    <td >Br-SAD</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/5T5A"><b> 5T5A</b></a> </td>
    <td >2</td>
    <td >2017</td>
    <td >Nat Chem Biol </td>
  </tr>
  <tr>
    <td >Hatchet</td>
    <td >The structure of Hatchet Ribozyme</td>
    <td >N/A</td>
    <td >MR</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/6JQ5"><b> 6JQ5</b></a> </td>
    <td >2</td>
    <td >2019</td>
    <td >Proc Natl Acad Sci USA</td>
  </tr>
  <tr>
    <td >Hatchet</td>
    <td >Hatchet Ribozyme Structure soaking with Ir(NH3)6+</td>
    <td >N/A</td>
    <td >Ir-SAD</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/6JQ6"><b> 6JQ6</b></a> </td>
    <td >2.63</td>
    <td >2019</td>
    <td >Proc Natl Acad Sci USA</td>
  </tr>
  <tr>
    <td >Pistol</td>
    <td >The native structure of native pistol ribozyme</td>
    <td >N/A</td>
    <td >MR</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/5K7C"><b> 5K7C</b></a> </td>
    <td >2.7</td>
    <td >2016</td>
    <td >Nat Chem Biol </td>
  </tr>
  <tr>
    <td >Pistol</td>
    <td >The structure of native pistol ribozyme, bound to Iridium</td>
    <td >N/A</td>
    <td >Ir-SAD</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/5K7D"><b> 5K7D</b></a> </td>
    <td >2.68</td>
    <td >2016</td>
    <td >Nat Chem Biol </td>
  </tr>
  <tr>
    <td >Pistol</td>
    <td >The structure of pistol ribozyme, soaked with Mn2+</td>
    <td >N/A</td>
    <td >MR</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/5K7E"><b> 5K7E</b></a> </td>
    <td >3.27</td>
    <td >2016</td>
    <td >Nat Chem Biol </td>
  </tr>
  <tr>
    <td >Pistol</td>
    <td >Crystal structure of Pistol, a class of self-cleaving ribozyme</td>
    <td >N/A</td>
    <td >Sm-MAD</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/5KTJ"><b> 5KTJ</b></a> </td>
    <td >2.97</td>
    <td >2017</td>
    <td >Proc Natl Acad Sci USA</td>
  </tr>
  <tr>
    <td >Pistol</td>
    <td >The structure of pistol ribozyme bound to magnesium</td>
    <td >N/A</td>
    <td >Br-SAD</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/6R47"><b> 6R47</b></a> </td>
    <td >3.1</td>
    <td >2019</td>
    <td >J Am Chem Soc</td>
  </tr>
  <tr>
    <td >Pistol</td>
    <td >Pistol ribozyme transition-state analog vanadate</td>
    <td >N/A</td>
    <td >MR</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/6UEY"><b> 6UEY</b></a> </td>
    <td >2.8</td>
    <td >2020</td>
    <td >Angew Chem Int Ed Engl</td>
  </tr>
  <tr>
    <td >Pistol</td>
    <td >Pistol ribozyme product crystal structure</td>
    <td >N/A</td>
    <td >MR</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/6UFJ"><b> 6UFJ</b></a> </td>
    <td >2.6</td>
    <td >2020</td>
    <td >Angew Chem Int Ed Engl</td>
  </tr>
  <tr>
    <td >Pistol</td>
    <td >Pistol ribozyme transition-state analog vanadate</td>
    <td >N/A</td>
    <td >MR</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/6UF1"><b> 6UF1</b></a> </td>
    <td >3.1</td>
    <td >2020</td>
    <td >Angew Chem Int Ed Engl</td>
  </tr>
  <tr>
    <td >Pistol</td>
    <td >Pistol ribozyme transition-state analog vanadate</td>
    <td >N/A</td>
    <td >MR</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/6UFK"><b> 6UFK</b></a> </td>
    <td >3.1</td>
    <td >2020</td>
    <td >Angew Chem Int Ed Engl</td>
  </tr>
  <tr>
    <td >Group I self-splicing intron</td>
    <td >Crystal structure of a self-splicing group Ⅰ intron with both exons</td>
    <td >Homo sapiens</td>
    <td >N/A</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/1U6B"><b> 1U6B</b></a> </td>
    <td >3.1</td>
    <td >2004</td>
    <td >Nature</td>
  </tr>
  <tr>
    <td >Group I self-splicing intron</td>
    <td >Structure of the Tetrahymena Ribozyme: Base Triple Sandwich and Metal Ion at the Active Site</td>
    <td >N/A</td>
    <td >N/A</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/1X8W"><b> 1X8W</b></a> </td>
    <td >3.8</td>
    <td >2004</td>
    <td >Mol Cell</td>
  </tr>
  <tr>
    <td >Group I self-splicing intron</td>
    <td >Crystal structure of an active group I ribozyme-product complex</td>
    <td >Staphylococcus virus Twort</td>
    <td >N/A</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/1Y0Q"><b> 1Y0Q</b></a> </td>
    <td >3.6</td>
    <td >2005</td>
    <td >Nat Struct Mol Biol</td>
  </tr>
  <tr>
    <td >Group I self-splicing intron</td>
    <td >Crystal structure of a group I intron/two exon complex that includes all catalytic metal ion ligands.</td>
    <td >Homo sapiens</td>
    <td >N/A</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/1ZZN"><b> 1ZZN</b></a> </td>
    <td >3.37</td>
    <td >2005</td>
    <td >Science</td>
  </tr>
  <tr>
    <td >Group I self-splicing intron</td>
    <td >Apo L-21 ScaI Tetrahymena ribozyme</td>
    <td >Tetrahymena thermophila</td>
    <td >N/A</td>
    <td >ELECTRON MICROSCOPY</td>
    <td ><a href="https://www.rcsb.org/structure/7EZ0"><b> 7EZ0</b></a> </td>
    <td >3.14</td>
    <td >2021</td>
    <td >Nature</td>
  </tr>
  <tr>
    <td >Group I self-splicing intron</td>
    <td >Crystal Structure of a Mutant (C109G,G212C) P4-P6 Domain of the Group I Intron from Tetrahymena Thermophilia</td>
    <td >N/A</td>
    <td >N/A</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/1L8V"><b> 1L8V</b></a> </td>
    <td >2.8</td>
    <td >2002</td>
    <td >Proc Natl Acad Sci U S A</td>
  </tr>
  <tr>
    <td >Group I self-splicing intron</td>
    <td >J4/5 Loop from the Candida albicans and Candida dubliniensis Group I Introns</td>
    <td >N/A</td>
    <td >N/A</td>
    <td >SOLUTION NMR</td>
    <td ><a href="https://www.rcsb.org/structure/1TUT"><b> 1TUT</b></a> </td>
    <td >N/A</td>
    <td >2004</td>
    <td >Biochemistry</td>
  </tr>
  <tr>
    <td >Group I self-splicing intron</td>
    <td >Group I self-splicing intron P4-P6 domain mutant U131A (with isopropanol soaking)</td>
    <td >Tetrahymena thermophila</td>
    <td >N/A</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/6BJX"><b> 6BJX</b></a> </td>
    <td >3.14</td>
    <td >2018</td>
    <td >Structure</td>
  </tr>
  <tr>
    <td >Group I self-splicing intron</td>
    <td >Group I self-splicing intron P4-P6 domain mutant U131A (without isopropanol soaking)</td>
    <td >Tetrahymena thermophila</td>
    <td >N/A</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/6D8L"><b> 6D8L</b></a> </td>
    <td >3.14</td>
    <td >2018</td>
    <td >Structure</td>
  </tr>
  <tr>
    <td >Group I self-splicing intron</td>
    <td >Group I self-splicing intron P4-P6 domain mutant A125U/G126U</td>
    <td >Tetrahymena thermophila</td>
    <td >N/A</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/6D8M"><b> 6D8M</b></a> </td>
    <td >3.7</td>
    <td >2018</td>
    <td >Structure</td>
  </tr>
  <tr>
    <td >Group I self-splicing intron</td>
    <td >Jak1 with compound 23</td>
    <td >Homo sapiens</td>
    <td >N/A</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/6DBN"><b> 6DBN</b></a> </td>
    <td >2.48</td>
    <td >2019</td>
    <td >J Med Chem</td>
  </tr>
  <tr>
    <td >Group I self-splicing intron</td>
    <td >Group I self-splicing intron P4-P6 domain mutant A230U</td>
    <td >Tetrahymena thermophila</td>
    <td >N/A</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/6D8O"><b> 6D8O</b></a> </td>
    <td >2.8</td>
    <td >2018</td>
    <td >Structure</td>
  </tr>
  <tr>
    <td >Lariat capping ribozyme</td>
    <td >Speciation of a group I intron into a lariat capping ribozyme</td>
    <td >Didymium iridis</td>
    <td >MAD-Ir</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/4P8Z"><b> 4P8Z</b></a> </td>
    <td >3.85</td>
    <td >2014</td>
    <td >PNAS</td>
  </tr>
  <tr>
    <td >Lariat capping ribozyme</td>
    <td >Speciation of a group I intron into a lariat capping ribozyme (Circularly permutated ribozyme)</td>
    <td >Didymium iridis</td>
    <td >MAD-Ir</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/4P95"><b> 4P95</b></a> </td>
    <td >2.5</td>
    <td >2014</td>
    <td >PNAS</td>
  </tr>
  <tr>
    <td >Lariat capping ribozyme</td>
    <td >peciation of a group I intron into a lariat capping ribozyme (Heavy atom derivative)</td>
    <td >Didymium iridis</td>
    <td >MAD-Ir</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/4P9R"><b> 4P9R</b></a> </td>
    <td >2.7</td>
    <td >2014</td>
    <td >PNAS</td>
  </tr>
  <tr>
    <td >Lariat capping ribozyme</td>
    <td >Lariat-capping ribozyme (circular permutation form)</td>
    <td >Didymium iridis</td>
    <td >MAD-Ir</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/6GYV"><b> 6GYV</b></a> </td>
    <td >2.5</td>
    <td >2014</td>
    <td >PNAS</td>
  </tr>
  <tr>
    <td >Lariat capping ribozyme</td>
    <td >Lariat-capping ribozyme with a shortened DP2 stem loop</td>
    <td >Didymium iridis</td>
    <td >N/A</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/6G7Z"><b> 6G7Z</b></a> </td>
    <td >3.3</td>
    <td >2019</td>
    <td >RNA</td>
  </tr>
  <tr>
    <td >Group II self-splicing intron</td>
    <td >Crystal Structure of a RNA Molecule Containing Domain 5 and 6 of the Yeast ai5g Group II Self-splicing Intron</td>
    <td >N/A</td>
    <td >N/A</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/1KXK"><b> 1KXK</b></a> </td>
    <td >3</td>
    <td >2002</td>
    <td >Science</td>
  </tr>
  <tr>
    <td >Group II self-splicing intron</td>
    <td >Crystal structure of a self-spliced group II intron</td>
    <td >N/A</td>
    <td >N/A</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/3BWP"><b> 3BWP</b></a> </td>
    <td >3.1</td>
    <td >2008</td>
    <td >Science</td>
  </tr>
  <tr>
    <td >Group II self-splicing intron</td>
    <td >Tertiary Architecture of the Oceanobacillus Iheyensis Group II Intron</td>
    <td >N/A</td>
    <td >N/A</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/3IGI"><b> 3IGI</b></a> </td>
    <td >3.125</td>
    <td >2009</td>
    <td >RNA</td>
  </tr>
  <tr>
    <td >Group II self-splicing intron</td>
    <td >Crystal structure of a eukaryotic group II intron lariat</td>
    <td >Pylaiella littoralis</td>
    <td >N/A</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/4R0D"><b> 4R0D</b></a> </td>
    <td >3.676</td>
    <td >2014</td>
    <td >Nature</td>
  </tr>
  <tr>
    <td >Group II self-splicing intron</td>
    <td >Structure of Oceanobacillus iheyensis group II intron in a ligand-free state in the presence of Tl+ and Mg2+</td>
    <td >Oceanobacillus iheyensis</td>
    <td >N/A</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/4E8Q"><b> 4E8Q</b></a> </td>
    <td >2.84</td>
    <td >2012</td>
    <td >Cell</td>
  </tr>
  <tr>
    <td >Group II self-splicing intron</td>
    <td >Oceanobacillus iheyensis group II intron domain 1 with iridium hexamine</td>
    <td >Oceanobacillus iheyensis HTE831</td>
    <td >N/A</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/4Y1N"><b> 4Y1N</b></a> </td>
    <td >3</td>
    <td >2015</td>
    <td >Nat Chem Biol </td>
  </tr>
  <tr>
    <td >Group II self-splicing intron</td>
    <td >Oceanobacillus iheyensis group II intron domain 1</td>
    <td >Oceanobacillus iheyensis HTE831</td>
    <td >N/A</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/4Y1O"><b> 4Y1O</b></a> </td>
    <td >2.95</td>
    <td >2015</td>
    <td >Nat Chem Biol </td>
  </tr>
  <tr>
    <td >Group II self-splicing intron</td>
    <td >Structure of the lariat form of a chimeric derivative of the Oceanobacillus iheyensis group II intron in the presence of NH4+ and MG2+.</td>
    <td >Oceanobacillus iheyensis</td>
    <td >N/A</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/5J01"><b> 5J01</b></a> </td>
    <td >3.39</td>
    <td >2016</td>
    <td >Science</td>
  </tr>
  <tr>
    <td >Group II self-splicing intron</td>
    <td >Structure a of Group II Intron Complexed with its Reverse Transcriptase</td>
    <td >Lactococcus lactis</td>
    <td >N/A</td>
    <td >ELECTRON MICROSCOPY</td>
    <td ><a href="https://www.rcsb.org/structure/5G2Y"><b> 5G2Y</b></a> </td>
    <td >4.5</td>
    <td >2016</td>
    <td >Nat Struct Mol Biol </td>
  </tr>
  <tr>
    <td >Group II self-splicing intron</td>
    <td >Structure of the lariat form of a chimeric derivative of the Oceanobacillus iheyensis group II intron in the presence of NH4+, MG2+ and an inactive 5' exon.</td>
    <td >Oceanobacillus iheyensis</td>
    <td >N/A</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/5J02"><b> 5J02</b></a> </td>
    <td >3.493</td>
    <td >2016</td>
    <td >Science</td>
  </tr>
  <tr>
    <td >Group II self-splicing intron</td>
    <td >Specific phosphorothioate substitution within domain 6 of a group II intron ribozyme leads to changes in local structure and metal ion binding</td>
    <td >Saccharomyces cerevisiae</td>
    <td >N/A</td>
    <td >SOLUTION NMR</td>
    <td ><a href="https://www.rcsb.org/structure/6EZ0"><b> 6EZ0</b></a> </td>
    <td >N/A</td>
    <td >2018</td>
    <td >J Biol Inorg Chem</td>
  </tr>
  <tr>
    <td >Group II self-splicing intron</td>
    <td >Structure of a group II intron retroelement after DNA integration</td>
    <td >Thermosynechococcus vestitus ,Thermosynechococcus vestitus BP-1</td>
    <td >N/A</td>
    <td >ELECTRON MICROSCOPY</td>
    <td ><a href="https://www.rcsb.org/structure/6MEC"><b> 6MEC</b></a> </td>
    <td >3.6</td>
    <td >2019</td>
    <td >Cell</td>
  </tr>
  <tr>
    <td >Bacterial RNase P</td>
    <td >RNase P protein from Bacillus subtilis</td>
    <td >Bacillus subtilis</td>
    <td >MIR</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/1A6F"><b> 1A6F</b></a> </td>
    <td >2.6</td>
    <td >1998</td>
    <td >Science</td>
  </tr>
  <tr>
    <td >Bacterial RNase P</td>
    <td >RNase P protein from Thermotoga maritima</td>
    <td >Thermotoga maritima</td>
    <td >SAD-Se</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/1NZ0"><b> 1NZ0</b></a> </td>
    <td >1.20 </td>
    <td >2003</td>
    <td >Proc Natl Acad Sci U S A</td>
  </tr>
  <tr>
    <td >Bacterial RNase P</td>
    <td >Crystal structure of the specificity domain of Ribonuclease P RNA</td>
    <td > Bacillus subtilis</td>
    <td >MAD-Pb</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/1NBS"><b> 1NBS</b></a> </td>
    <td >3.15</td>
    <td >2003</td>
    <td >Nature</td>
  </tr>
  <tr>
    <td >Bacterial RNase P</td>
    <td >Crystal structure of the specificity domain of Ribonuclease P of the A-type</td>
    <td >N/A</td>
    <td >SAD-Ba</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/1U9S"><b> 1U9S</b></a> </td>
    <td >2.9</td>
    <td >2004</td>
    <td >Science</td>
  </tr>
  <tr>
    <td >Bacterial RNase P</td>
    <td >Crystal structure of the RNA subunit of Ribonuclease P. Bacterial A-type</td>
    <td >N/A</td>
    <td >SIRAS</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/2A2E"><b> 2A2E</b></a> </td>
    <td >3.85</td>
    <td >2005</td>
    <td >Nature </td>
  </tr>
  <tr>
    <td >Bacterial RNase P</td>
    <td >Crystal Structure of Bacterial Ribonuclease P RNA</td>
    <td >N/A</td>
    <td >MAD-Os</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/2A64"><b> 2A64</b></a> </td>
    <td >3.30 </td>
    <td >2005</td>
    <td >Proc Natl Acad Sci U S A</td>
  </tr>
  <tr>
    <td >Bacterial RNase P</td>
    <td >Mapping metal-binding sites in the catalytic domain of bacterial RNase P RNA</td>
    <td >N/A</td>
    <td >MR</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/3DHS"><b> 3DHS</b></a> </td>
    <td >3.6</td>
    <td >2009</td>
    <td > RNA</td>
  </tr>
  <tr>
    <td >Bacterial RNase P</td>
    <td >Structure of a Bacterial Ribonuclease P Holoenzyme in Complex with tRNA</td>
    <td >Thermotoga maritima</td>
    <td >MR / MAD / MIRAS</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/3Q1Q"><b> 3Q1Q</b></a> </td>
    <td >3.80 </td>
    <td >2010</td>
    <td >Nature</td>
  </tr>
  <tr>
    <td >Bacterial RNase P</td>
    <td >Crystal structure of a bacterial RNase P holoenzyme in complex with TRNA and in the presence of 5' leader</td>
    <td >Thermotoga maritima</td>
    <td >MR / MAD / MIRAS</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/3Q1R"><b> 3Q1R</b></a> </td>
    <td >4.21 </td>
    <td >2010</td>
    <td >Nature</td>
  </tr>
  <tr>
    <td >Archaeal  RNase P</td>
    <td >Crystal structure analysis of protein component Ph1496p of P.horikoshii ribonuclease P</td>
    <td >Pyrococcus horikoshii</td>
    <td >MR</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/2CZW"><b> 2CZW</b></a> </td>
    <td >1.90 </td>
    <td >2006</td>
    <td >Biochem Biophys Res Commun</td>
  </tr>
  <tr>
    <td >Archaeal  RNase P</td>
    <td >cryo-EM structure of an archaeal Ribonuclease P</td>
    <td >Methanocaldococcus jannaschii DSM 2661, Methanocaldococcus jannaschii</td>
    <td >N/A</td>
    <td >Electron microscopy</td>
    <td ><a href="https://www.rcsb.org/structure/6K0A"><b> 6K0A</b></a> </td>
    <td >4.6</td>
    <td >2019</td>
    <td >Nat Commun</td>
  </tr>
  <tr>
    <td >Archaeal  RNase P</td>
    <td >cryo-EM structure of archaeal Ribonuclease P with mature tRNA</td>
    <td >Methanocaldococcus jannaschii DSM 2661, Escherichia coli, Methanocaldococcus jannaschii</td>
    <td >N/A</td>
    <td >Electron microscopy</td>
    <td ><a href="https://www.rcsb.org/structure/6K0B"><b> 6K0B</b></a> </td>
    <td >4.3</td>
    <td >2019</td>
    <td >Nat Commun</td>
  </tr>
  <tr>
    <td >Nuclear RNase P</td>
    <td >Cryo-EM structure of yeast Ribonuclease P</td>
    <td >Saccharomyces cerevisiae S288C</td>
    <td >N/A</td>
    <td >Electron microscopy</td>
    <td ><a href="https://www.rcsb.org/structure/6AGB"><b> 6AGB</b></a> </td>
    <td >3.48 </td>
    <td >2018</td>
    <td >Science</td>
  </tr>
  <tr>
    <td >Nuclear RNase P</td>
    <td >Cryo-EM structure of yeast Ribonuclease P with pre-tRNA substrate</td>
    <td >Saccharomyces cerevisiae S288C</td>
    <td >N/A</td>
    <td >Electron microscopy</td>
    <td ><a href="https://www.rcsb.org/structure/6AH3"><b> 6AH3</b></a> </td>
    <td >3.48</td>
    <td >2018</td>
    <td >Science</td>
  </tr>
  <tr>
    <td >Nuclear RNase P</td>
    <td >Cryo-EM structure of human Ribonuclease P</td>
    <td >Homo sapiens</td>
    <td >N/A</td>
    <td >Electron microscopy</td>
    <td ><a href="https://www.rcsb.org/structure/6AHR"><b> 6AHR</b></a> </td>
    <td >3.92 </td>
    <td >2018</td>
    <td >Cell</td>
  </tr>
  <tr>
    <td >Nuclear RNase P</td>
    <td >Cryo-EM structure of human Ribonuclease P with mature tRNA</td>
    <td >Homo sapiens</td>
    <td >N/A</td>
    <td >Electron microscopy</td>
    <td ><a href="https://www.rcsb.org/structure/6AHU"><b> 6AHU</b></a> </td>
    <td >3.66</td>
    <td >2018</td>
    <td >Cell</td>
  </tr>
  <tr>
    <td >Nuclear RNase P</td>
    <td >Crystal structure of human RPP40</td>
    <td >Homo sapiens</td>
    <td >SAD-Se</td>
    <td >X-ray diffraction</td>
    <td ><a href="https://www.rcsb.org/structure/6AHV"><b> 6AHV</b></a> </td>
    <td >2.6</td>
    <td >2018</td>
    <td >Cell</td>
  </tr>
  <tr>
    <td >RNase MRP</td>
    <td >Structure of yeast RNase MRP holoenzyme</td>
    <td >Saccharomyces cerevisiae S288C</td>
    <td >N/A</td>
    <td >Electron microscopy</td>
    <td ><a href="https://www.rcsb.org/structure/6W6V"><b> 6W6V</b></a> </td>
    <td >3.00 </td>
    <td >2020</td>
    <td >Nat Commun </td>
  </tr>
  <tr>
    <td >RNase MRP</td>
    <td >Cryo-EM structure of yeast Ribonuclease MRP</td>
    <td >Saccharomyces cerevisiae S288C</td>
    <td >N/A</td>
    <td >Electron microscopy</td>
    <td ><a href="https://www.rcsb.org/structure/7C79"><b> 7C79</b></a> </td>
    <td >2.5</td>
    <td >2020</td>
    <td >Science</td>
  </tr>
  <tr>
    <td >RNase MRP</td>
    <td >Cryo-EM structure of yeast Ribonuclease MRP with substrate ITS1</td>
    <td >Saccharomyces cerevisiae S288C, Saccharomyces cerevisiae</td>
    <td >N/A</td>
    <td >Electron microscopy</td>
    <td ><a href="https://www.rcsb.org/structure/7C7A"><b> 7C7A</b></a> </td>
    <td >2.8</td>
    <td >2020</td>
    <td >Science</td>
  </tr>
</tbody>